spec/requests/notifications_spec.rb

```diff
@@ -44,7 +46,7 @@ def renders_comments_html(comment)
       it "renders the signup page" do
         get "/notifications"
 
-        expect(response.body).to include("Continue with")
+        expect(response.body).to include("OR")
```

Comment:
It seems very vague that we're testing for OR. Same for tests below.

Reply:
I agree, hard to tell at a glance what appears different between a logged-in and logged-out user

Reply:
Here and everywhere else we have used `OR` because earlier it was either using `Continue with` or a full text like `Have a password? Continue with your email address`. Both of these represent that we are on `Login` page. To make things less confusing I have now used `By signing in` which also represents `Login` page.

Let me know if this can be improved further.